### ISSUE TITLE
undefined method `assets' in RC10

### ISSUE THREAD

**Issue by NiklasHae** [unpoly/unpoly#180]
Upgrading from rc9 to rc10 raises the following exception in a project without sprockets (e.g. a project that relies on webpacker for all assets):

```
4: from /persistent/bundle/ruby/2.6.0/gems/railties-5.2.4.4/lib/rails/initializable.rb:61:in `block in run_initializers'
3: from /persistent/bundle/ruby/2.6.0/gems/railties-5.2.4.4/lib/rails/initializable.rb:32:in `run'
2: from /persistent/bundle/ruby/2.6.0/gems/railties-5.2.4.4/lib/rails/initializable.rb:32:in `instance_exec'
1: from /persistent/bundle/ruby/2.6.0/gems/unpoly-rails-2.0.0.pre.rc10/lib/unpoly/rails/engine.rb:23:in `block in <class:Engine>'
/persistent/bundle/ruby/2.6.0/gems/railties-5.2.4.4/lib/rails/railtie/configuration.rb:97:in `method_missing': undefined method `assets' for #<Rails::Application::Configuration:0x00005564182f1b70> (NoMethodError)
```

Easily reproducable:

```
rails new rails_without_sprockets --skip-sprockets
bundle add unpoly
bundle install
rails c
```

If you require `sprockets/railtie` in `application.rb` it works flawlessly again.

RC10 adds something to the `assets` config namespace, which is normally only defined if sprockets is required.
A project relying completely on webpacker does not have this namespace.
https://github.com/unpoly/unpoly/blob/master/lib/unpoly/rails/engine.rb#L20-L23

Idea for a fix:

```ruby
if app.config.respond_to? :assets # the assets config namespace could be missing (e.g. if sprockets is not configured for a project)
  if is_local_gem
    app.config.assets.paths << source_folder.join('javascripts').to_s
    app.config.assets.paths << source_folder.join('stylesheets').to_s
  else
    app.config.assets.paths << dist_folder.to_s
  end
end
```

Other gems put this behind a config flag, that can be toggled manually in an initializer, maybe that's a better idea.

**Comment by adam12** [comment]
I think this is fixed by c2db67475b8298453e0ae3315c8ac97bc8666d04 in rc11 (just released).

**Comment by triskweline** [comment]
It is :)
